jsonpaths/com.snowplowanalytics.snowplow.enrichments/iab_spiders_and_robots_enrichment_1.json

```diff
@@ -0,0 +1,24 @@
+{
+    "jsonpaths": [
+        "$.schema.vendor",
+        "$.schema.name",
+        "$.schema.format",
+        "$.schema.version",
+        "$.hierarchy.rootId",
+        "$.hierarchy.rootTstamp",
+        "$.hierarchy.refRoot",
+        "$.hierarchy.refTree",
+        "$.hierarchy.refParent",
+        "$.data.enabled",
+        "$.data.name",
+        "$.data.parameters.excludeUseragentFile.database",
+        "$.data.parameters.excludeUseragentFile.uri",
+        "$.data.parameters.includeUseragentFile.database",
+        "$.data.parameters.includeUseragentFile.uri",
+        "$.data.parameters.ipFile.database",
+        "$.data.parameters.ipFile.uri",
+        "$.data.vendor",
+        "$.data.parameters.httpPassword",
+        "$.data.parameters.httpUsername"
+    ]
+}
```

Comment:
I don't think this and the associated sql are needed btw

Reply:
Agree